pkg/middlewares/aws_eip_association_filter.go

```diff
@@ -0,0 +1,47 @@
+package middlewares
+
+import (
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/aws"
+)
+
+/**
+  Fetching eip association from remote return every association but some of them are embedded in eip.
+  This middleware will check for every eip_association that here is no corresponding association_id inside eip.
```

Comment:
Our classic approach was to expand resources instead of filtering, any particular reason to have choose another approach ?

Reply:
I took the ticket @wbeuil already check what we could do so I took his advice. I didn't check if we have all we need to expand in this case.